<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="group_templates_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QTextBrowser" name="text_open_file">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="combobox_deconv_algorithm">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>MSAlign</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="group_templates_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Delta Mass parameters</string>
     </property>
     <widget class="QLabel" name="label_most_deff_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>From</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_deltamass_mass_min">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_most_deff_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_most_att_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Da</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_deltamass_mass_max">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>1,500</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="button_load_data">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Load 
 Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_open_file">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_templates_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>771</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Plot Mass Difference</string>
    </property>
    <widget class="QLabel" name="label_most_deff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Bin size</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Show</string>
     </property>
     <widget class="QCheckBox" name="checkbox_deltamass_countintensity">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Count x Intensity</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox_deltamass_intensity">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Intensity</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox_deltamass_count">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Annotation</string>
     </property>
     <widget class="QCheckBox" name="checkbox_annotation_show">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Show </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_massdifference_annotation_height">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_massdifference_annotation_distance">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_most_deff_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_most_deff_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="button_plot_histogram">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Plot 
 Histograms</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="text_massdifference_bin_size">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string>0.01</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_export_histogram_data">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_show_peaklist">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Peaks</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_show_possible_modifications">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Mods</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Results"/>
    <addaction name="actionOpen_Results"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Results">
   <property name="text">
    <string>Save Results</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Results">
   <property name="text">
    <string>Open Results</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_load_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_plot_histogram</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plot_histogram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_export_histogram_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_histogram_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_show_possible_modifications</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_possible_modifications()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_show_peaklist</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_peaklist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_open_file</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Results</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_results()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Results</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_results()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_data()</slot>
  <slot>plot_histogram()</slot>
  <slot>export_histogram_data()</slot>
  <slot>export_rt_shift_data()</slot>
  <slot>show_possible_modifications()</slot>
  <slot>show_peaklist()</slot>
  <slot>open_file()</slot>
  <slot>save_results()</slot>
  <slot>open_results()</slot>
 </slots>
</ui>
